### DISCUSSION TITLE
Storage Error: RPC failure, probably timeout.

### DISCUSSION THREAD

**Discussion by Desh1975** [vesoft-inc/nebula#5599] [Q&A]
Hello ALL,
I get after update Nebula DB to version  NebulaGraph Studio: v3.7.0 following error.
(NebulaGraph for Docker Desktop Extension 0.4.8, NebulaGraph: v3.5.0, NebulaGraph Studio: v3.7.0)

**Storage Error: RPC failure, probably timeout.**

It's always same error if make insert or query on very small structure.

Thanks in advance for cooperation,

**Comment by wey-gu** [comment]
Dear @Desh1975 

Could you see anything interesting in logs?

```
ls -l ~/.nebulagraph/logs/*/*.ERROR
```

**Reply by alibek-sumup** [reply]
@wey-gu    there is no such folder .nebulagraph

**Reply by wey-gu** [reply]
It should be there on the host OS of the docker desktop, if it's a Windows, the ~/ is the home-dir for your user, too.
